doc/Makefile

```diff
@@ -33,15 +33,17 @@ all: man html pages
 man:
 	rm -rf man
 	mkdir -p man
-	$(OPAM) $(HELPFMT) > man/opam.1
+	$(OPAM) $(HELPFMT) > man/opam.1 2> man/err
 	for i in $(TOPICS); do\
-	  $(OPAM) $$i $(HELPFMT) > man/opam-$$i.1;\
+	  $(OPAM) $$i $(HELPFMT) > man/opam-$$i.1 2>> man/err ; \
 	done
-	$(OPAM) admin $(HELPFMT) > man/opam-admin.1
+	$(OPAM) admin $(HELPFMT) > man/opam-admin.1 2>> man/err
 	for i in $(TOPICS_ADMIN); do\
-	  $(OPAM) admin $$i $(HELPFMT) > man/opam-admin-$$i.1;\
+	  $(OPAM) admin $$i $(HELPFMT) > man/opam-admin-$$i.1 2>> man/err ; \
 	done
-	$(OPAM_INSTALLER) $(HELPFMT) > man/opam-installer.1
+	$(OPAM_INSTALLER) $(HELPFMT) > man/opam-installer.1 2>> man/err
+# Dune doesn't (yet) support --no-print-directory
```

Comment:
```suggestion
# TODO Dune will support --no-print-directory in 1.7.0 and higher
```

Reply:
🙂 At the time I pushed this branch, I had only just opened the PR in Dune and didn't want to presume until it was merged. This comment could be changed on master, but I don't think we want to constrain Dune to >= 1.7.0 on the 2.0 branch

Reply:
Makes sense!